TruckMuncher/VendorMapViewController.swift

```diff
@@ -196,16 +197,45 @@ class VendorMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     }
     
     func requestServingMode (isServing: Bool) {
+        var itemsOutOfStock = getOutOfStockItems()
+        if itemsOutOfStock.count > 0 && isServing == true {
+            var message = String()
+            for item in itemsOutOfStock {
+                message += "- " + (item as RMenuItem).name + "\n"
```

Comment:
SmokeShack is one example of this. They sell out of their entire menu multiple days per week. Although they are a restaurant and not a food truck so that's not a perfect example. This could happen at bigger events or at festivals milwaukee has when the food trucks are getting a lot of business.